CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [2.25.1]
 
+## Added
+
+- Ongoing product displayed on the syllabus are now link to the order
+  details page in the learner dashboard.
+
```

Comment:
IMO this is a fix as a link was displayed but a wrong one.

Reply:
It's both a fix and a add:
* when we had an active enrollment, the wrong link was displayed.
* when we had an order but no enrollment, we didn't had link.